docs/news.rst

```diff
@@ -3,6 +3,42 @@
 Release notes
 =============
 
+Scrapy 1.3.1 (2017-02-XX)
+-------------------------
+
+New features
+~~~~~~~~~~~~
+
+- Support ``'True'`` and ``'False'`` string values for boolean settings (:issue:`2519`);
+  you can now do something like ``scrapy crawl myspider -s REDIRECT_ENABLED=False``.
+- Support kwargs with ``response.xpath()`` to use :ref:`XPath variables <topics-selectors-xpath-variables>`
+  and ad-hoc namespaces declarations ;
+  this requires at least Parsel v1.1 (:issue:`2457`).
+- Add support for Python 3.6 (:issue:`2485`).
```

Comment:
It seems the linked ticket is more about running tests in Python 3.6; there are still unresolved issues like https://github.com/scrapy/scrapy/issues/2511